src/components/RoleSelectOption/RoleSelectOption.tsx

```diff
@@ -36,60 +36,63 @@ const optionWrapperStyles = css`
   border-radius: 6px;
 `
 
-export const Option: React.FC<OptionProps> = ({ label, text, src, alt }) => {
+export const Option: React.FC<OptionProps> = ({ label, text }) => {
   return (
     <RoleSelectWrapper>
       <div
-        className={
-          label === 'Buyer'
-            ? 'inner-select-wrapper-buyer'
-            : 'inner-select-wrapper-seller'
-        }>
+        className="inner-select-wrapper">
```

Comment:
we are using styled components so we should not be assigning class names.
Create separate styled component for each div/class name